funnel/templates/forms.html.jinja2

```diff
@@ -211,34 +219,51 @@
     </script>
   {%- endif %}
 {%- endmacro %}
-{%- macro recaptcha(formId, forWrapperId) %}
-  <script type="text/javascript">
-    var formId = "{{ formId }}";
-    var forWrapperId = "{{ forWrapperId }}";
-    window.onInvisibleRecaptchaSubmit = function (recaptcha_response) {
-      var postUrl = $('#' + formId).attr('action');
-      var onSuccess = function(responseData) {
-        $('#' + forWrapperId).html(responseData);
+{%- macro recaptcha(formId, ajax=false, forWrapperId='') %}
+  {%- if ajax %}
+    <script type="text/javascript">
+      var formId = "{{ formId }}";
```

Comment:
Use `var formId = {{ formId|tojson }};`. Not using any filter will cause the default HTML escaping filter to be applied.